guides/source/engines.md

```diff
@@ -1515,12 +1515,12 @@ To hook into the initialization process of one of the following classes use the
 
 These are the available configuration hooks. They do not hook into any particular framework, instead they run in context of the entire application.
 
-| Hook                   | Use Case                                                                              |
-| ---------------------- | ------------------------------------------------------------------------------------- |
-| `before_configuration` | First configurable block to run. Called before any initializers are run.              |
-| `before_initialize`    | Second configurable block to run. Called before frameworks initialize.                |
-| `before_eager_load`    | Third configurable block to run. Does not run if `config.cache_classes` set to false. |
-| `after_initialize`     | Last configurable block to run. Called after frameworks initialize.                   |
+| Hook                   | Use Case                                             |
+| ---------------------- | ---------------------------------------------------- |
+| `before_configuration` | Called before any initializers are run.              |
+| `before_initialize`    | Called before frameworks initialize.                 |
+| `before_eager_load`    | Does not run if `config.cache_classes` set to false. |
```

Comment:
The explanation is gathered from the rdoc.

https://github.com/rails/rails/blob/71b4bf698f7443bd8d2e436bc9f37b4d6ec8fc8d/railties/lib/rails/railtie/configuration.rb#L53-L72

I don't think the explanation is useless.
If triggered order is removed, at least it is not explained when `before_eager_load` block to run.